### PR TITLE
Solution to Code Challenges

### PR DESCRIPTION
**Pull request by nanaowusunyamekye**

### CODE REVIEW

File: longestString/longestString.js [hunk 1]
```diff
@@ -1,4 +1,19 @@
...
  * Return the longest string in the array.
- */
\ No newline at end of file
+ */
+
+const longestStringInArray = myArray => {
+    let lenth = 0;
+    let longestString;
+
+    for (var i = 0; i < myArray.length; i++) {
```
Comment (ryan-hamblin) [inline longestString/longestString.js]: You're using Es6 everywhere else? Why not use let vs. var here?

**Review by ryan-hamblin — COMMENTED** [review]
This is great! A classic solution to this challenge!

File: longestString/longestString.js (hunk 1 above)
Comment (nanaowusunyamekye) [reply longestString/longestString.js]: Sure.Thanks

File: reverseCase/reverseCase.js [hunk 2]
```diff
@@ -3,3 +3,18 @@
  * Example: 'Hello World' -> 'hELLO wORLD'
  * Assume that each string will contain only spaces and letters.
  */
+
+ const caseChanger = (str) => {
+     let transformed = "";
```
Comment (ryan-hamblin) [inline reverseCase/reverseCase.js:8]: Well done Sir! You're doing great. I think this is a very clean easy-to-read solution

File: binarySearch/binarySearch.js [hunk 3]
```diff
@@ -11,5 +11,30 @@
 **/
 
 const binarySearch = (nums, target) => {
+    let min = 0;
+	let max = nums.length - 1;
```
Comment (ryan-hamblin) [inline binarySearch/binarySearch.js:15]: Watch your indentation here.

File: binarySearch/binarySearch.js [hunk 4]
```diff
@@ -11,5 +11,30 @@
...
+    let midnum = Math.floor((min + max) / 2);
+    let found = -1;
+
+    while(min < max) {
```
Comment (ryan-hamblin) [inline binarySearch/binarySearch.js:19]: This is a solid iterative approach! Nicely done!

File: evenOccurrences/evenOccurrences.js [hunk 5]
```diff
@@ -12,4 +12,15 @@
 
 const evenOccurence = (arr) => {
   // Your code here.
+
+    let sortArray = arr.sort(); // sort the array
+    let occurer; // this will hold the value that will be caught when the for loop runs
+
+    for(let i = 0 ;  i < sortArray.length; i++){
+      if(sortArray[i] === sortArray[i+1]){
+         occurer = sortArray[i]
```
Comment (ryan-hamblin) [inline evenOccurrences/evenOccurrences.js:21]: Missing semi-colon

File: largestPrimePalindrome/largestPrimePalindrome.js [hunk 6]
```diff
@@ -4,4 +4,31 @@
...
+    //  Create function to check Prime
+     const checkPrime = (number) => {
+     for (let i = 2; i < number; i++) if (number % i === 0) return false;
+     return true;
+
+     if (number < 2) return false;
+   };
+    // Create function to check if number is Palindrome
+   checkPalindrome = (n) => {
+   for (let x = n; x >= 0; x--) {
```
Comment (ryan-hamblin) [inline largestPrimePalindrome/largestPrimePalindrome.js:23]: This for loop block should be indented

**Review by ryan-hamblin — COMMENTED** [review]
Simple formatting issues I've been noticing, but other than that things look good. I'd like to see a finished solution for the CallBack practice stuff!

File: binarySearch/binarySearch.js (hunk 3 above)
Comment (nanaowusunyamekye) [reply binarySearch/binarySearch.js:15]: Will do that. Still not really solid on callbacks. Will read, practice more and re-submit the 'CallBack Practice' challenge for review. Thanks for the feedback

Comment (ryan-hamblin) [reply binarySearch/binarySearch.js:15]: Would you like to do some work with me on them tomorrow at some point? I'm working in the morning tomorrow and off for the weekend until next Tuesday.

**Pull request closed without merging**